hydra-configs-torch/tests/test_register_optimizers.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from hydra.core.config_store import ConfigStore
+import hydra_configs.torch.optim
+
+cs = ConfigStore()
+
+# registers all 'base' optimizer configs with configstore instance
+hydra_configs.torch.optim.register()
```

Comment:
the code defines the function register_configs().

Reply:
thanks, I changed the name to `register()` but missed committing the latest changes when i added the test